packages/gatsby-source-contentful/src/normalize.js

```diff
@@ -193,14 +193,13 @@ function prepareTextNode(node, key, text, createNodeId) {
   return textNode
 }
 
-function prepareStructuredTextNode(node, key, content, createNodeId) {
+function prepareRichTextNode(node, key, content, createNodeId) {
   const str = stringify(content)
-  const structuredTextNode = {
+  const richTextNode = {
     ...content,
     id: createNodeId(`${node.id}${key}RichTextNode`),
     parent: node.id,
     children: [],
-    [key]: str,
```

Comment:
Could this be potentially considered as breaking change? Is there harm it keeping (just for backward compat if someone for some reason used it). If `key` is the same as field defined in `extendNodeType` then `extendNodeType` will win, so there shouldn't be harm in keeping it

Reply:
You're right, this is a breaking change; however, we can leave it outside of the scope of this PR and introduce this breaking change later. Would you recommend reverting this change?

Reply:
Sorry for late reply - if leaving this in doesn't mess anything up, then I would recommend it. If it's possible to avoid breaking change, we should always do it so users don't need to do extra work migrate their code.